ansible_collections/arista/avd/molecule/eos_designs_negative_unit_tests/create.yml

```diff
@@ -1,6 +1,6 @@
 ---
 - name: Configure local folders
-  hosts: EOS_DESIGNS_FAILURES
+  hosts: EOS_DESIGNS_DEPRECATED_VARS
```

Comment:
Why s this changed? this is the negative_unit scenario so this should not change

Reply:
oops! my mistake

Reply:
Fixed